public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+
+:marked
+  The `takeJob()` method invokes the `next()` method of `onJobTaken`, and passes the `hero` assigned to this `HeroPanel`. 
+  The parameter of `next()` is the argument of the event emitted. Thanks to the strong type notation in TypeScript, `EventEmitter` is a generic type,
+  so we could declare `onJobTaken` as an `EventEmitter<Hero>`. When we invoke `next`, the TypeScript compiler checks whether the event argument 
+  is compatible with `Hero`.
+  
+  To use `@Output()` and `EventEmitter`, `HeroPanel` needs to import them:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-panel.ts', 'import', null, {blk: /(Output|EventEmitter)/g})
+
+:marked
+  Now our `HeroPanel` is ready to emit its event. `HeroJobBoard` attaches its `heroTakesJob()` event handler method to the `onJobTaken` output
+  event of `HeroPanel` using the output property binding syntax &mdash; wrapping the snake-case name of the output property, `on-job-taken`, into parentheses.
+  The output property binding expression passes the `$event` identifier &mdash; that represents the argument of the emitted event &mdash; to the handler method:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-job-board.ts', 'component', 'take-job-event/hero-job-board.ts', {pnk: /(\(on-job-taken\))/g, blk: /(heroTakesJob\(hero: Hero\))/g})
+
+.alert.is-critical
+  :marked
+    Although this technique works in many situations, it's an **antipattern**, and Angular discourages using it.
+  
+:marked
+  The event argument is the `hero` property value of `HeroPanel` that is passed to the `next()` method of the `EventEmitter` instance beyond
+  `onJobTaken`. `HeroJobBoard` receives the `Hero` instance that undertakes the job, and adds it to its `respondingHeroes` array.
+  
+  With output event binding, we established this communication structure from `HeroPanel` children to `HeroJobBoard`:
+  
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/child-to-parent.png" alt="Output event binding")
+  
++makeExample('component-communication/ts/fragments/hero-job-board-with-parentref.ts', null, 'take-job-parentref/hero-job-board.ts (extract)', {pnk: /(\[parent\]|jobBoard\(\))/g})
+
+:marked
+  `HeroPanel` now has a reference to its parent, and it can easily call the `heroTakesJob()` method:
+  
++makeExample('component-communication/ts/fragments/hero-panel-with-parentref.ts', null, 'take-job-parentref/hero-panel.ts (extract)', {blk: /(HeroJobBoard)/g, pnk: /(parent|takeJob\(\)|heroTakesJob)/g})
+
+:marked
+  Now, we can invite heroes, announce job requests, and our heroes can opt to undertake the job. The `HeroJobBoard` component lists responding
+  heroes in the UI:
+  
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/take-job-ui.png" alt="Hero undertakes job")
+  
+:marked
+  ### Using `@Output`
```

Comment:
I believe you have a sequence problem. The discussion of `@Output` belongs up with the event emitter property.

When you remove the `parent` example above, this discussion will fall into place.